app/views/govuk_publishing_components/components/docs/intervention.yml

```diff
@@ -38,6 +38,17 @@ examples:
       suggestion_link_text: "You can now apply for a permit online."
       suggestion_link_url: "/permit"
 
+  open_suggestion_link_in_new_tab:
+    description: |
+      For example if you wanted to drive users to another online task, you don't want to completely hijack
+      their original flow on GOV.UK. Adding the option to open links in a new tab helps to address this.
+      Link text should tell the user that the link opens in a new tab.
+      Note: "(opens in a new tab)" is added to link text if the phrase isn't included.
```

Comment:
Are these newlines meant to render as newlines? You need to put an empty line between each one if that's the case, not sure if intended or not.

Reply:
Intended - wanted them viewable within the a certain length in the source. Makes no difference as you say.